ui-components/build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
     api "androidx.constraintlayout:constraintlayout-compose:1.0.1"
     api "com.google.android.material:compose-theme-adapter:${libraries.composeTheme}"
     implementation "androidx.compose.material3:material3:${libraries.material3}"
+    api "com.google.accompanist:accompanist-themeadapter-material3:0.28.0"
```

Comment:
please, can you check if there is already one dependencies in `dependencies.gradle`, if not create it. thanks!